src/commands/project/delete/source.ts

```diff
@@ -189,6 +191,29 @@ export class Source extends SfCommand<DeleteSourceJson> {
         fsPaths: await getPackageDirs(),
         include: this.componentSet,
       });
+      // Confirm if the user wants to delete GenAiPlugins as part of an agent
+      if (!this.flags['no-prompt']) {
```

Comment:
so if `--no-prompt` it will delete all genAiFuncs?

Reply:
No, only GenAiPlugins.  GenAiFuncs are not included in the delete.